setup.py

```diff
@@ -26,8 +26,8 @@
     # specify dependencies
     install_requires=[
         "setuptools",
-        "django-oscar<2.0",
```

Comment:
We want to keep oscar 1.6 compatibility if possible, so lets keep this unspecified.

Reply:
Done.